pkg/operator/starter.go

```diff
@@ -99,6 +99,7 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 	targetConfigReconciler := NewTargetConfigReconciler(
 		ctx,
 		os.Getenv("RELATED_IMAGE_OPERAND_IMAGE"),
+		os.Getenv("RELATED_IMAGE_SOFTTAINTER_IMAGE"),
```

Comment:
This requires to update bundle.Dockerfile with:
```
ARG SOFTTAINER_IMAGE=registry.stage.redhat.io/kube-descheduler-operator/kube-descheduler-rhel9-operator@sha256:426baca9e36c7833d335262a7b8f73c318cd749ca7767726fc86109948a44d96
ARG REPLACED_SOFTTAINER_IMG=quay.io/openshift/origin-cluster-kube-descheduler-operator:latest
```
and set `RELATED_IMAGE_SOFTTAINTER_IMAGE` under the csv file (in the deployment definition the same way as in `deploy/05_deployment.yaml`).

Reply:
done